lib/linguist/languages.yml

```diff
@@ -5088,6 +5080,25 @@ Standard ML:
   codemirror_mode: mllike
   codemirror_mime_type: text/x-ocaml
   language_id: 357
+Starlark:
+  type: programming
+  tm_scope: source.python
+  ace_mode: python
+  codemirror_mode: python
+  codemirror_mime_type: text/x-python
+  color: "#76d275"
+  extensions:
+  - ".bzl"
+  filenames:
+  - BUCK
+  - BUILD
+  - BUILD.bazel
+  - WORKSPACE
+  - WORKSPACE.bazel
```

Comment:
Not sure if I mentioned that before, but there's only a [single such file](https://github.com/search?q=filename%3AWORKSPACE.bazel) on GitHub. It's not enough to warrant addition.

Reply:
@pchaigno, [yep, you did](https://github.com/github/linguist/pull/4759#discussion_r362743336). I was under an impression that’s OK to include it and monitor the usage. Or should I remove it and it will be added later if monitoring results will be fine?

Reply:
> Or should I remove it and it will be added later if monitoring results will be fine?

Yes, that's how we usually proceed.

Reply:
Fair enough. Removed, thanks!